lib/teracy-dev/util.rb

```diff
@@ -215,11 +215,20 @@ def self.override(originHash, sourceHash)
                 id_existing = false
                 originHash[key] ||= []
                 originHash[key].each do |val1|
+
+                  if val1['_id_deprecated'] != nil && val['_id'] == val1['_id_deprecated']
+                    @@logger.warn("The _id: #{val1['_id_deprecated']} is deprecated, use the _id: #{val1['_id']} instead")
```

Comment:
@@logger.warn("The _id: '#{val1['_id_deprecated']}' is deprecated, use the _id: '#{val1['_id']}' instead for #{val}")

Reply:
something wrong, I expect 1 time of the warning only, got 2 times:

```
$ vagrant status
[TeracyDev::Util][WARN]: The _id: '0' is deprecated, use the _id: 'kernel-core' instead for {"_id"=>"0", "location"=>{"git"=>{"remote"=>{"origin"=>"https://github.com/teracyhq-incubator/teracy-dev-core.git"}, "branch"=>"develop"}}, "require_version"=>">= 0.4.0-SNAPSHOT", "enabled"=>true}
/Users/hoatle/teracy-dev/lib/teracy-dev/util.rb:220:in `block (3 levels) in override'

[TeracyDev::Util][WARN]: The _id: '0' is deprecated, use the _id: 'kernel-core' instead for {"_id"=>"0", "location"=>{"git"=>{"remote"=>{"origin"=>"https://github.com/teracyhq-incubator/teracy-dev-core.git"}, "branch"=>"develop"}}, "require_version"=>">= 0.4.0-SNAPSHOT", "enabled"=>true}
/Users/hoatle/teracy-dev/lib/teracy-dev/util.rb:220:in `block (3 levels) in override'
```